src/Batch/Batch.Test/ScenarioTests/BatchAccountTests.ps1

```diff
@@ -145,4 +149,31 @@ function Test-CreateNewBatchAccountWithNoPublicIp
     {
         Remove-AzResourceGroup $resourceGroup
     }
+}
+
+<#
+.SYNOPSIS
+Tests creating an account with identity set to Microsoft.KeyVault
+#>
+function Test-CreateNewBatchAccountWithSystemIdentity
+{
+    # Setup
+    $accountName = Get-BatchAccountName
+    $resourceGroup = Get-ResourceGroupName
+
+    try
+    {
+        $location = Get-BatchAccountProviderLocation
+        # Create a Batch account
+        New-AzResourceGroup -Name $resourceGroup -Location $location
+        $createdAccount = New-AzBatchAccount -Name $accountName -ResourceGroupName $resourceGroup -Location $location -Identity @{Type = "SystemAssigned"}
```

Comment:
Possibly consider just making the argument `-IdentityType` which takes in a string?